lib/permissions.js

```diff
@@ -17,44 +23,70 @@ const SERVICES = {
   speech: 'kTCCServiceSpeechRecognition',
 };
 
-const STATUS_UNSET = 'unset';
-const STATUS_YES = 'yes';
-const STATUS_NO = 'no';
-
-
 function toInternalServiceName (serviceName) {
   if (_.has(SERVICES, _.toLower(serviceName))) {
     return SERVICES[_.toLower(serviceName)];
   }
   throw new Error(`'${serviceName}' is unknown. Only the following service names are supported: ${JSON.stringify(_.keys(SERVICES))}`);
 }
 
-function assertStatus (status) {
-  const supportedValues = [STATUS_UNSET, STATUS_NO, STATUS_YES];
-  if (!supportedValues.includes(status)) {
-    throw new Error(`Status value '${status}' is unknown. Only the following values are supported: ${JSON.stringify(supportedValues)}`);
+function formatStatus (status) {
+  return [STATUS_UNSET, STATUS_NO].includes(status) ? _.toUpper(status) : status;
+}
+
+async function execWix (args) {
+  try {
+    await fs.which(WIX);
+  } catch (e) {
+    throw new Error(`${WIX} binary has not been found in your PATH. ` +
+      `Please install it ('brew tap wix/brew && brew install wix/brew/applesimutils') to ` +
+      `be able to change application permissions`);
+  }
+
+  log.debug(`Executing ${WIX} with arguments ${JSON.stringify(args)}`);
```

Comment:
Maybe we could do something like:

````log.debug(`Executing "`${WIX} ${shellParse.quote(args)}`"`)```

It's handy being able to cut-and-paste shell commands and test them in the terminal.

Reply:
Added shell.quote